container-images/kolla/horizon/extend_start.sh

```diff
@@ -80,6 +80,18 @@ function config_octavia_dashboard {
         "${SITE_PACKAGES}/openstack_dashboard/local/enabled/_1482_project_load_balancer_panel.py"
 }
 
+function config_watcher_dashboard {
+    for file in ${SITE_PACKAGES}/watcher_dashboard/local/enabled/_*[^__].py; do
+        config_dashboard "${ENABLE_WATCHER}" \
+            "${SITE_PACKAGES}/watcher_dashboard/local/enabled/${file##*/}" \
+            "${SITE_PACKAGES}/openstack_dashboard/local/enabled/${file##*/}"
+    done
+
+    config_dashboard "${ENABLE_WATCHER}" \
+        "${SITE_PACKAGES}/watcher_dashboard/conf/watcher_policy.json" \
+        "/etc/openstack-dashboard/watcher_policy.json"
+}
+
```

Comment:
I'm afraid of breaking horizon deployment downstream by adding this as we need ENABLE_WATCHER as this script requires ENABLE_WATCHER variable which we need to add in horizon-operator and also requires files which are installed by openstack-watcher-ui which will not be installed downstream. What if we add the dashboard configuration part for a future PR or at least don't run it so far?

Reply:
@amoralej Yes, indeed, it will break. I am moving this whole piece. https://issues.redhat.com/browse/OSPRH-11277 will track the same, will add it in future pr.